jvm/src/main/java/com/cloudera/tsexamples/JavaStocks.java

```diff
@@ -80,12 +86,14 @@ public static void main(String[] args) {
         (Vector x) -> TimeSeriesStatisticalTests.dwtest(x)
     );
 
-    System.out.println(dwStats.min(
-        (Tuple2<String, Double> a, Tuple2<String, Double> b) -> a._2.compareTo(b._2)
-    ));
-    System.out.println(dwStats.max(
-        (Tuple2<String, Double> a, Tuple2<String, Double> b) -> a._2.compareTo(b._2)
-    ));
+    class StatsComparator implements Comparator<Tuple2<String,Double>>, java.io.Serializable {
+        public int compare(Tuple2<String,Double> a, Tuple2<String,Double> b) {
```

Comment:
Would you mind adding a space after the commas to be in line with the code style>